packages/extension-ui/src/Popup/AuthManagement/DisconnectApp.tsx

```diff
@@ -55,8 +54,9 @@ function DisconnectApp({ className }: Props): React.ReactElement<Props> {
       />
       <div className={className}>
         <div className='content'>
-          <Svg
+          <img
             className='animated-remove-icon'
+            key={Math.floor(Math.random() * Date.now())}
```

Comment:
Date.now() is unique timestamp in milliseconds - should be enough without extra Math floor/random